corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -88,7 +88,7 @@ def _copy_unprocessed_forms(self):
             _migrate_form_attachments(sql_form, couch_form)
             _migrate_form_operations(sql_form, couch_form)
 
-            diffs = json_diff(couch_form.to_json(), sql_form.to_json())
+            diffs = json_diff(couch_form.to_json(), sql_form.to_json(), track_list_indices=False)
```

Comment:
so that you can compare list diffs without caring about the list index where the diff occurred